### PR TITLE
Fix python snippet typo on getting started page

### PR DESCRIPTION
**Pull request by toshitanian**
This PR fixes the python script typo in the documentation.

https://google.github.io/mediapipe/getting_started/python_framework.html#graph

### CODE REVIEW

File: docs/getting_started/python_framework.md [hunk 1]
```diff
@@ -249,7 +249,7 @@ three stages: initialization and setup, graph run, and graph shutdown.
     graph.start_run()
 
     graph.add_packet_to_input_stream(
-        'in_stream', mp.packet_creator.create_str('abc').at(0))
+        'in_stream', mp.packet_creator.create_string('abc').at(0))
```
Comment (toshitanian) [inline docs/getting_started/python_framework.md:252]: Correct method name is `create_string`.

File: docs/getting_started/python_framework.md [hunk 2]
```diff
@@ -249,12 +249,12 @@ three stages: initialization and setup, graph run, and graph shutdown.
...
```
Comment (toshitanian) [inline docs/getting_started/python_framework.md:252]: Correct method name is create_string.

File: docs/getting_started/python_framework.md [hunk 3]
```diff
@@ -249,12 +249,12 @@ three stages: initialization and setup, graph run, and graph shutdown.
...
 
     rgb_img = cv2.cvtColor(cv2.imread('/path/to/your/image.png'), cv2.COLOR_BGR2RGB)
     graph.add_packet_to_input_stream(
         'in_stream',
-        mp.packet_creator.create_image_frame(format=mp.ImageFormat.SRGB,
+        mp.packet_creator.create_image_frame(image_format=mp.ImageFormat.SRGB,
```
Comment (toshitanian) [inline docs/getting_started/python_framework.md:257]: correct parameter name is `image_format`
https://github.com/google/mediapipe/blob/a9b643e0f5978425d948050a2228398929733d05/mediapipe/python/packet_creator.py#L55

**Comment by jiuqiant** [comment]
Thanks for reporting this. We also fixed it internally and it's released in https://github.com/google/mediapipe/commit/e6c19885c6d3c6f410c730952aeed2852790d306.

**Pull request closed without merging**